test/include/dlaf_test/matrix/util_generic_blas_extensions.h

```diff
@@ -0,0 +1,69 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
```

Comment:
Please fix the copyright year on the files you added.
```suggestion
// Copyright (c) 2018-2024, ETH Zurich
```